libc/test/src/math/smoke/CMakeLists.txt

```
@@ -4154,3 +4154,55 @@ add_fp_unittest(
     libc.src.__support.FPUtil.basic_operations
     libc.src.math.f16mulf128
 )
+
+add_fp_unittest(
+  daddl
+  SUITE
+    libc-math-smoke-tests
+  SRCS
+    daddl_test.cpp
+  HDRS
+    AddTest.h
+  DEPENDS
+    libc.src.math.daddl
+    libc.src.__support.FPUtil.generic.add_sub
```

The smoke tests themselves don't use `libc.src.__support.FPUtil.generic.add_sub`.

```suggestion
  DEPENDS
    libc.src.math.daddl
```